<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewWindow</class>
 <widget class="QMainWindow" name="NewWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>950</width>
    <height>744</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="LabelComboBox">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>20</y>
      <width>251</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>30</y>
      <width>191</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QListView" name="listView">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>290</y>
      <width>351</width>
      <height>291</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="LabelPriests">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>240</y>
      <width>56</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ButtonAddPriest">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>600</y>
      <width>101</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Priest</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ButtonDeletePriest">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>600</y>
      <width>101</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete Priest</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ButtonChangeParish">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>600</y>
      <width>101</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Change Parish</string>
    </property>
   </widget>
   <widget class="QListView" name="listView_2">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>390</y>
      <width>256</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="LabelParishName">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>120</y>
      <width>141</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Nazwa Parafii</string>
    </property>
   </widget>
   <widget class="QLabel" name="LabelEQ">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>350</y>
      <width>56</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>60</y>
      <width>291</width>
      <height>241</height>
     </rect>
    </property>
    <property name="text">
     <string>Losowe Zdjecie</string>
    </property>
   </widget>
   <widget class="QLabel" name="LabelBudget">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>610</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="LabelShop">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>320</y>
      <width>151</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_2">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>390</y>
      <width>261</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="ButtonBuy">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>550</y>
      <width>80</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>BUY</string>
    </property>
   </widget>
   <widget class="QLabel" name="LabelMoney">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>620</y>
      <width>56</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>$</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>950</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <action name="actionDodaj_Parafie">
   <property name="text">
    <string>Create New Parish</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
